src/ec/packages/ec-component-site-header-harmonised/site-header-harmonised.story.js.bckp

```diff
@@ -45,4 +47,19 @@ storiesOf('Components/Site Headers/Harmonised', module)
     {
       notes: { markdown: notes, json: dataGroup2 },
     }
+  )
+  .add(
+    'group 3',
+    () =>
+      siteHeaderHarmonised(
+        merge(dataGroup3, {
+          logo: {
+            src:
+              'https://inno-ecl.s3.amazonaws.com/media/examples/placeholder.svg',
+          },
+        })
+      ),
+    {
+      notes: { markdown: notes },
```

Comment:
I think we can have the "hardcoded" notes replaced by the real data used to render the component as we do for the other two groups, no..? I refer to the additional parameter in the notes object, this is what we use to pass the real data to the storybook panel.